src/app/categories/[categoryId]/pairwise-ranking/page.tsx

```diff
@@ -48,13 +58,10 @@ const CategoryPairwiseRankingPage = () => {
 	console.log('PairwiseData', pairwisePairs);
 	const [firstProject, secondProject] = pairwisePairs?.data.pairs[0] ?? [];
 
-	const currentPercentage =
-		((pairwisePairs?.data.votedPairs ?? 0) /
-			(pairwisePairs?.data.totalPairs ?? 1)) *
-		100;
-	const thresholdPercentage = pairwisePairs?.data.threshold
-		? +pairwisePairs?.data.threshold * 100
-		: 100;
+	const threshold = pairwisePairs?.data.threshold ?? 0;
+	const votedPairs = pairwisePairs?.data.votedPairs ?? 0;
+	const totalPairs = pairwisePairs?.data.totalPairs ?? 1;
+	const progressPercentage = (votedPairs / totalPairs / threshold) * 100;
```

Comment:
Let’s add a check to prevent divide by 0 in case totalPairs is 0  (if pairwisePairs?.data.totalPairs is 0)